src/Modules/OnPremisesDataGateway/Commands.OnPremisesDataGateway/MicrosoftPowerBIMgmt.OnPremisesDataGateway.psd1

```diff
@@ -0,0 +1,121 @@
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'Microsoft.PowerBI.Commands.OnPremisesDataGateway.dll'
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = '0492de38-4b0e-4c99-b3f3-1ab7e908a955'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Power BI PowerShell - On-Premises Data Gateway cmdlets for Microsoft Power BI'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '3.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+# RequiredModules = @()
+
+# Assemblies that must be loaded prior to importing this module
+# RequiredAssemblies = @()
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+# FormatsToProcess = @()
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+# NestedModules = @()
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = '*'
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'Add-OnPremisesDataGatewayClusterUser', 'Get-OnPremisesDataGatewayCluster', 'Get-OnPremisesDataGatewayClusterStatus', 'Get-OnPremisesDataGatewayInstallers', 'Get-OnPremisesDataGatewayTenantPolicy', 'Remove-OnPremisesDataGatewayCluster', 'Remove-OnPremisesDataGatewayClusterMember','Remove-OnPremisesDataGatewayClusterMember', 'Set-OnPremisesDataGatewayCluster', 'Set-OnPremisesDataGatewayInstallers', 'Set-OnPremisesDataGatewayTenantPolicy'
```

Comment:
If some of these cmdlets depend on unreleased APIs we might just want to comment them out for now till the APIs are live